From 9cf2c47c796f6a74f65823027977b9f161986fde Mon Sep 17 00:00:00 2001
From: wqyoung <quanyang.wang@windriver.com>
Date: Sun, 30 Oct 2016 17:03:14 +0800
Subject: [PATCH 1/2] video: mxsfb: delete redundant kfree functions in
 resources release

In commit "release the timimgs after init fb information", it add the routine
"display_timings_release" to release the resources timing, and there is no need
to call kfree to release timing again. So delete the redundant function kfree
function and reserve the function "display_timings_release".

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 drivers/video/fbdev/mxsfb.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/drivers/video/fbdev/mxsfb.c b/drivers/video/fbdev/mxsfb.c
index 579be80..740e0b2 100644
--- a/drivers/video/fbdev/mxsfb.c
+++ b/drivers/video/fbdev/mxsfb.c
@@ -1199,10 +1199,9 @@ static int mxsfb_init_fbinfo_dt(struct mxsfb_info *host)
 
 put_timings_node:
 	of_node_put(timings_np);
-	display_timings_release(timings);
 put_display_node:
 	if (timings)
-		kfree(timings);
+		display_timings_release(timings);
 	of_node_put(display_np);
 	return ret;
 }
-- 
1.7.5.4

